package.json

```diff
@@ -91,6 +91,8 @@
     "@playwright/test": "1.38.0",
     "@qlik/eslint-config": "0.4.18",
     "@qlik/prettier-config": "0.3.0",
+    "@scriptappy/cli": "^0.10.0",
```

Comment:
Pin versions

Reply:
Cannot believe you have to keep pointing this out 😊 
Maybe there is a way to have them pinned when installed actually, I will have a look around. Thanks!

Reply:
Maybe when you install an explicit version they get pinned? But yeah that would be nice to have since it easy to forget

Reply:
And of course it exists https://classic.yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add-exact-e

Reply:
Even better https://github.com/yarnpkg/yarn/issues/3045